doc/templates/index.html

```diff
@@ -183,6 +183,7 @@ <h4 class="sk-landing-call-header">Community</h4>
         <li><strong>Questions?</strong> See <a href="faq.html">FAQ</a> and <a href="https://stackoverflow.com/questions/tagged/scikit-learn">stackoverflow</a></li>
         <li><strong>Mailing list:</strong> <a href="https://mail.python.org/mailman/listinfo/scikit-learn">scikit-learn@python.org</a></li>
         <li><strong>Gitter:</strong> <a href="https://gitter.im/scikit-learn/scikit-learn">gitter.im/scikit-learn</a></li>
+        <li>All communication on all channels should respect <a href="https://www.python.org/psf/conduct/">PSF's code of conduct</a></li>
```

Comment:
Maybe "Communication on all channel" to be less redundant?

Also needs a . at the end.

Reply:
channel -> channels :)